src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Map/ChangeBasemap/ChangeBasemap.xaml.cs

```diff
@@ -18,50 +19,31 @@ namespace ArcGISRuntime.WPF.Samples.ChangeBasemap
         name: "Change basemap",
         category: "Map",
         description: "Change a map's basemap. A basemap is beneath all layers on a `Map` and is used to provide visual reference for the operational layers.",
-        instructions: "Use the drop down menu to select the active basemap from the list of available basemaps.",
+        instructions: "When the basemap gallery appears, select a basemap to be displayed.",
         tags: new[] { "basemap", "map" })]
     public partial class ChangeBasemap
     {
-        // Dictionary that associates names with basemaps.
-        private readonly Dictionary<string, Basemap> _basemapOptions = new Dictionary<string, Basemap>()
-        {
-            {"Streets", new Basemap(BasemapStyle.ArcGISStreets)},
-            {"Streets - Night", new Basemap(BasemapStyle.ArcGISStreetsNight)},
-            {"Imagery", new Basemap(BasemapStyle.ArcGISImageryStandard)},
-            {"Imagery with Labels", new Basemap(BasemapStyle.ArcGISImagery)},
-            {"Dark Gray Canvas", new Basemap(BasemapStyle.ArcGISDarkGray)},
-            {"Light Gray Canvas", new Basemap(BasemapStyle.ArcGISLightGray)},
-            {"Navigation", new Basemap(BasemapStyle.ArcGISNavigation)},
-            {"OpenStreetMap", new Basemap(BasemapStyle.OSMStandard)}
-        };
-
         public ChangeBasemap()
         {
             InitializeComponent();
 
-            // Create the UI, setup the control references and execute initialization
+            // Create the UI, setup the control references and execute initialization.
             Initialize();
```

Comment:
```suggestion
            _ = Initialize();
```
Needed for the warning.